### DISCUSSION TITLE
blog/how-to-write-blog

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#27] [Announcements]
# blog/how-to-write-blog

我们本应从写博客这项活动中获益甚大，正如从任何良好的写作活动中获益甚大。在这篇文章中，我打算认真对待博客，看看“如何写好博客”这个主题有什么话可说。

https://pathos.page/blog/how-to-write-blog

**Comment by soulizer** [comment]
博客首先是为自己而写的，博主用写博客的方式来记录自己的想法或观点从模糊到清晰、从笼统到具体的过程，同时尽量让自己的描述简短、通俗易懂，以便于读者也能从中受益。这样写出来的文章，绝对是上乘之作。

但在实际上，许多博客只是博主用于分享自己的见闻感想、表达自己既有观点的平台，往往难以“利己”；同时，如果博主没有在优化表达方面下太多功夫，“利他”的效果也较为有限。此时的博客仿佛一个“高级版QQ空间”。

**Comment by JanYork** [comment]
博主这个主题是自己写的吗

**Reply by Pathsis** [reply]
不是我自己写的。主题来源是[Lonor](https://lawrenceli.me/blog/how)。我做了一些修改。

**Comment by shuibaco** [comment]
其实吧，我以前也是这么想的。但是实践之后给我整不会写了，还因此“断更”了两年。后来我才意识到写博客是融入我生活（生命）里的东西，能写下去就已经占用了不少有限的意志力，没有多余的精力也没必要去保证文章的“有用性”。

每个人都有自己的一套生活原则，我很欣赏你的原则和想法。其实底层逻辑也是相通的，想要做好一件事，无非就是做到极致，只不过方向不同罢了🤝

**Reply by Pathsis** [reply]
看到你的评论，我觉得对那些致力于**记录**的博客有点苛刻啦哈哈

**Reply by qlibp** [reply]
文章的"有用性"对于每个人而言并不一致，对于本博主而言，博客的作用是一种思想练习工具。用于记录生活的博客，对于其博主而言自然也是有作用的，最起码能起到抒发情绪，保持精神健康的作用。

我个人很认可作者此文的结论，把激发自身思考作为博客的目标，促进我们长期思考，对于个人而言才是一个更重要的意义。博客自身带有的公共性迫使我们检验自己的思考逻辑，是相比于日记而言更有益的思想载体。

**Comment by tumutanzi** [comment]
说得非常好，非常认同，也一直是这样追这样的原则来做。

**Comment by fishhh666** [comment]
我现在的困扰在于如何提高自己的词汇水平，经常想到一个形容词但褒贬不合适，或者程度不合适，试过问ai，但效果很不好。还有句式上很乱，想找一找这种类型的书，而不是那种讲文笔教写作的书，不知道从何找起。不知道博主有没有推荐的~
至于内容我倒跟博主想法不太一样，日常随性类型的记录对于人生也很有价值啊，QQ和微博的那年今日我超级爱看，而博客也是我更表达自我的东西。

**Comment by lshcool** [comment]
写得很棒，
建议很好。
但我还是喜欢写日记。

**Comment by yydschen** [comment]
大佬！能分享下你修改的paperesque这个主题吗？感觉非常简洁比较喜欢～
